Signal/src/ViewControllers/PhotoLibrary/ImagePickerController.swift

```diff
@@ -26,6 +26,11 @@ class ImagePickerGridController: UICollectionViewController, PhotoLibraryDelegat
 
     private let titleLabel = UILabel()
 
+    private var selectedIds = Set<String>()
```

Comment:
There's an interesting question about the ordering of the selected assets.  Order of selection?  Order within the collection?